client/packages.go

```diff
@@ -37,6 +39,9 @@ type Package struct {
 	Version       string `json:"version"`
 }
 
+// Packages are collections of Package items
```

Comment:
I hear what you're saying, however [`run-checks` runs Golint](https://github.com/ubuntu-core/snappy/blob/master/run-checks#L105) so any exported types must have a Godoc comment.